app/lib/screens/history/history.dart

```diff
@@ -30,23 +27,8 @@ class TravelHistory extends StatelessWidget {
         },
       ),
       floatingActionButton: FloatingActionButton(
-        onPressed: () async {
-          try {
-            buildPdf();
-            savePdf();
-            final output = await getTemporaryDirectory();
-            String fullPath = ("${output.path}/example.pdf");
-
-            Navigator.push(
-                context,
-                MaterialPageRoute(
-                    builder: (context) => PdfPreview(
-                          path: fullPath,
-                        )));
-          } on HttpException catch (err) {
-            print('Error: $err');
-          }
-        },
+        onPressed: () => Navigator.push(
+            context, MaterialPageRoute(builder: (context) => PdfPreview())),
```

Comment:
In the future, we can further refactor  the navigation by using onGenerateroute feature in flutter. We can even pass data as named parameters which is even more cleaner. Thanks kelly!

Reply:
I believe the named parameters require a mapping of name to widget somewhere, is that right?